ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -28,6 +28,19 @@
 - include: gen_certs.yml
   when: inventory_hostname == groups['masters'][0]
 
+- name: Archive certificates
```

Comment:
Can you walk me through the logic here? I can understand tar'ing the certs from the 1st master, then having add'l masters pull. Or, have the tar pushed to the ansible controller from master 0, so add'l masters get the tar pushed to them from the ansible controller.